### ISSUE TITLE
MD037/no-space-in-emphasis triggers when using LaTex inline math formula

### ISSUE THREAD

**Issue by MalcolmMielle** [DavidAnson/vscode-markdownlint#219]
Similar to #133 , MD032/blanks-around-lists: Lists should be surrounded by blank lines is triggered for inline formulas such as 

```markdown
A **reward** $R_t$ is a scalar feedback signal---if you have conflicting goals, you _need_ a scale on which to compare goals to be able to translate them to a scalar feedback. It indicates how well an agent is doing at step _t_.
```

**Comment by DavidAnson** [comment]
I don't see that with the following test file in VS Code using `markdownlint` extension version 0.45.0:

```markdown
# Issue 219

A **reward** $R_t$ is a scalar feedback signal---if you have conflicting goals, you _need_ a scale on which to compare goals to be able to translate them to a scalar feedback. It indicates how well an agent is doing at step _t_.
```

Instead, I see two instances of MD037/no-space-in-emphasis. Both of these should be gone in the next version of the library as seen here: https://dlaa.me/markdownlint/#%25m%23%20Issue%20219%0A%0AA%20**reward**%20%24R_t%24%20is%20a%20scalar%20feedback%20signal---if%20you%20have%20conflicting%20goals%2C%20you%20_need_%20a%20scale%20on%20which%20to%20compare%20goals%20to%20be%20able%20to%20translate%20them%20to%20a%20scalar%20feedback.%20It%20indicates%20how%20well%20an%20agent%20is%20doing%20at%20step%20_t_.%0A

Could you please double-check your example?

**Comment by MalcolmMielle** [comment]
Gosh my brain, you're right, I reported the wrong error -_-. It's indeed `MD037/no-space-in-emphasis` instead of `MD032/blanks-around-lists`. I'll correct the title, and I'm happy to know it'll be fixed in the next version.

Sorry again for the confusion, it's late where I am :S.

**Comment by DavidAnson** [comment]
No worries! :)
